features/steps/amun_api.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Thoth's integration tests
+# Copyright(C) 2020 Bissenbay Dauletbayev
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Amun integration tests for Thoth deployment."""
+
+import os
+import requests
+import time
+from datetime import timedelta
+
+from amun.swagger_client import InspectionApi
+from amun.swagger_client import InspectionSpecification
+from amun.swagger_client import InspectionSpecificationBuild
+from amun.swagger_client import InspectionSpecificationBuildRequests
+from amun.swagger_client import InspectionSpecificationBuildRequestsHardware
+from amun.swagger_client import InspectionSpecificationRun
+from amun.swagger_client import InspectionSpecificationRunRequests
+from amun.swagger_client import InspectionSpecificationRunRequestsHardware
+from amun.swagger_client import Configuration
+from amun.swagger_client import ApiClient
+
+from amun import get_inspection_status
+from amun import get_inspection_build_log
+
+from behave import given, when, then
+
+
+@given("amun service is accessible using {scheme}")
```

Comment:
Why are we not reusing basic method? https://github.com/thoth-station/integration-tests/blob/87c4c4a80c0fd299cdba8a80b1d2b3f44e5f71f3/features/steps/basic.py#L31

Reply:
Good point, I remember I suggested that idea to @goern and we decided to move with separate methods because the method in `basic` only checks for `user-api` if it is up and running. In the beginning, we wanted to check if `amun-api` service is running inside this method, however, we agreed that as `user-api` and `amun-api` are separate services and they do not depend on each other.